src/lib/cfa/ntp_conf.rb

```diff
@@ -125,10 +125,17 @@ def each(&block)
         record_entries.each(&block)
       end
 
+      # Get last record in collection
+      def last
+        record_entries.last
+      end
+
       # Adds a new Record object to the collection.
+      # @note it creates new instance of record after this.
```

Comment:
maybe I can rephrase it like `@note argument is not member of collection and instead new instance is created from its augeas content`